docs/contributing/doing.md

```diff
@@ -0,0 +1,141 @@
+
+[GitHub Labels](https://developer.github.com/v3/issues/labels/) are used to manage issues broadly considered work items grouped by color and prefixed by label name.
+
+## Creating branches
+
+Create a unique branch name tracking `origin/master` to perform your work.
+
+```bash
+git checkout -b <users/alias/feature-branch-name> origin/master
+```
+
+- Use all lower cased hyphen-delimited branch names for readability
+- Format as <users/alias/branch-name>. For example, users/jdoe/task-issue-title
+- Create as new branch for each work issue using the `-b` argument.
+- Commit messages should use lowercase imperative present tense. Use commands. Instead of “I added tests for ...” use “add tests for ...”
+
+## Pull requests
+
+When landing a pull request (PR), individuals must modify the original commit message to include comprehensive details. This detail along with other git history meta information deploys as part of the documentation changelog history. Changlogs are generated by [Lerna](https://lernajs.io/) using [Conventional commits](https://conventionalcommits.org/).
+
+PR comments should include the issue number per the [github closing issue keywords](https://help.github.com/articles/closing-issues-using-keywords/) documentation.
+
+One or more collaborators will automatically be assigned to review each PR based on areas of [ownership](https://github.com/Microsoft/fast-dna/blob/master/.github/CODEOWNERS).
+
+**`<type>`** is required to be at least one of the following:
+
+- chore: A change that does not impact distributed packages
+- feat: Adding, updating, or removing a feature
+- fix: A bug fix.
+
+**`<description>`** is required and speaks to what the user gets from this PR:
+
+- Be concise
+- Use lowercase
+- Use imperative, present tense (e.g. `add` not `adds`)
+- Do not end with a period. Period.
+- Avoid redundant words
+
+Accepted title format
+
+```comment
+<type>: <description>
+```
+
+Example titles
+
+```comment
+feat: add a border radius to MSFT button
+chore: add unit-tests for all components
+fix: update MSFT design system to use 3px border radius
+```
+
+### Formatting messages
+
+Optionally, include more detail in the message.
+
+`<body>` is used to provide clarity and context to the description.
+
+`<footer>` is used to provide additional metadata about the pull request such as issue fixing (fix #19, close #19, resolve #19). Additionally, is used to denote breaking changes.
+
+Accepted message format
+
+```comment
+<body>
+
+<footer>
+```
+
+Example message
+
+```comment
+Sequence indicators markup is now located after the slider markup to at the top for improved accessibility because the screen reader can read all of the slide options individually before interacting with the slider.
+
+closes #213
+
+BREAKING CHANGE : Describe [what and why](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#specification) this change was required.
```

Comment:
No space before ":",

see other PR for my comments on when this can be added to the message

Reply:
I did see them and I modified it so it was easy to read in GitHub Issue template.

Reply:
removed